airflow/logging_backend.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from airflow import configuration
+from airflow.logging_backends.base_logging_backend import BaseLoggingBackend
+from airflow.utils.imports import import_class_by_name
+
+LOGGING_BACKEND = None
+
+# Backend alias to full class name mapping
+LOGGING_BACKEND_ALIAS = {
+    'elasticsearch': ('airflow.logging_backends.elasticsearch_logging_backend'
+                      '.ElasticsearchLoggingBackend'),
+}
+
+
+def cached_logging_backend():
```

Comment:
`_get_logging_backend` is a helper function that make the logic in `cached_logging_backend` more clean. It follows the pattern of `cached_app` in Airflow : https://github.com/apache/incubator-airflow/blob/master/airflow/www/app.py#L158